src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/component/CommunityAPIClient.kt

```diff
@@ -18,9 +20,35 @@ class CommunityAPIClient(
       .retrieve()
       .bodyToMono(Unit::class.java)
       .onErrorResume { e ->
-        logger.error("Call to community api failed", e, StructuredArguments.kv("requestBody", requestBody))
+        handleResponse(e, requestBody)
         Mono.empty()
       }
       .subscribe()
   }
+
+  fun makeSyncPostRequest(uri: String, requestBody: Any) {
```

Comment:
i would expect a synchronous request method like this to return a value?

Reply:
Yes it does return an appointment id which I will accept and just log.